app/shared/db/prisma.server.ts

```diff
@@ -1,16 +1,14 @@
-import {
-  PrismaClient,
-  Prisma,
-  PrismaClientKnownRequestError,
-} from "@webstudio-is/sdk/lib/prisma.server";
+import { PrismaClient, Prisma } from "@prisma/client";
+import { PrismaClientKnownRequestError } from "@prisma/client/runtime";
 
-export const prisma = new PrismaClient();
-export { Prisma, PrismaClientKnownRequestError };
+declare global {
+  // allow global `var` declarations
+  // eslint-disable-next-line no-var
+  var prisma: PrismaClient | undefined;
+}
+
+export const prisma = global.prisma || new PrismaClient();
```

Comment:
is this fixing the dev issue with hot reload?

Reply:
needs at least an explanation

Reply:
cc @okonet seems like @SaraVieira fixed that problem you also mentioned

Reply:
Yes, the explanation is here: https://www.prisma.io/docs/guides/database/troubleshooting-orm/help-articles/nextjs-prisma-client-dev-practices

Its linked in the PR, want me to add it to the code?

Reply:
ah, yes please link it inline, because not obvious to the reader

Reply:
Done!